Add threading info (#1446)
fluent/fluent-bit-docs: pipeline/inputs/disk-io-metrics.md

The **disk** input plugin, gathers the information about the disk throughput of the running system every certain interval of time and reports them.

> The Disk I/O metrics plugin creates metrics that are log-based \(I.e. JSON payload\). If you are looking for Prometheus-based metrics please see the Node Exporter Metrics input plugin.

Reviewer: ```suggestion
The Disk I/O metrics plugin creates metrics that are log-based, such as JSON payload.
For Prometheus-based metrics, see the Node Exporter Metrics input plugin.
```
